openacademy/tests/test_openacademy_course.py

```diff
@@ -0,0 +1,80 @@
+from . commontest import CommonTest
+
+from psycopg2.errors import CheckViolation, UniqueViolation
+from odoo.tools import mute_logger
+
+
+class TestCourse(CommonTest):
+
+    def setUp(self, *args, **kwargs):
+        super().setUp(*args, **kwargs)
```

Comment:
Superfluous method, only calls super. It may be removed

Reply:
Do you mean `super().setUp()` ?

Reply:
Okay, I understand, sorry.